olx_importer/management/commands/load_course_data.py

```diff
@@ -0,0 +1,200 @@
+        longest_identifier_len = 0
+        print("Reading static assets...\n")
+
+        for file_path in file_paths:
+            identifier = str(file_path.relative_to(course_data_path))
+            longest_identifier_len = max(longest_identifier_len, len(str(identifier)))
+
+            data_bytes = file_path.read_bytes()
+
+            num_files += 1
+            cum_size += len(data_bytes)
+            print(f"Static file #{num_files}: ({(cum_size / 1_000_000):.2f} MB)", end="\r")
+
+            data_hash = create_hash_digest(data_bytes)
+            if file_path.suffix == "":
+                mime_type = "text/plain"
+            else:
+                mime_type, _encoding = mimetypes.guess_type(identifier)
+
+            mime_types_seen[mime_type] += 1
+
+            if mime_type is None:
+                print(identifier)
+
+            item_raw_id = item_raw_id_cache.get((mime_type, data_hash))
+            if item_raw_id is None:
+                item_raw, _created = ItemRaw.objects.get_or_create(
+                    learning_context=self.learning_context,
+                    mime_type=mime_type,
+                    hash_digest=data_hash,
+                    defaults={
+                        'data': data_bytes,
+                        'size': len(data_bytes),
+                        'created': now,
+                    }
+                )
+                item_raw_id = item_raw.id
+
+            item_info, _created = ItemInfo.objects.get_or_create(
+                learning_context=self.learning_context,
+                type='static_asset',
+                item_raw_id=item_raw_id,
+                defaults={'created': now},
+            )
+            paths_to_item_raw_ids[identifier] = item_raw_id
+
+        print(f"{num_files} assets, totaling {(cum_size / 1_000_000):.2f} MB")
+        print(f"Longest identifier length seen: {longest_identifier_len}")
+        print("MIME types seen:")
+        for mime_type_str, freq in sorted(mime_types_seen.items()):
+            print(f"* {mime_type_str}: {freq}")
+
+        return paths_to_item_raw_ids
+
+
+    def import_block_type(self, block_type, content_path, static_asset_paths_to_atom_ids, item_raw_id_cache, now):
+        items_found = 0
+
+        # Find everything that looks like a reference to a static file appearing
+        # in attribute quotes, stripping off the querystring at the end. This is
+        # not fool-proof as it will match static file references that are
+        # outside of tag declarations as well.
+        static_files_regex = r"""['"]\/static\/(.+?)["'\?]"""
+
+        for xml_file_path in content_path.iterdir():
+            items_found += 1
+            identifier = xml_file_path.stem
+            data_bytes = xml_file_path.read_bytes()
+            hash_digest = create_hash_digest(data_bytes)
+            data_str = codecs.decode(data_bytes, 'utf-8')
+            mime_type = f'application/vnd.openedx.xblock.{block_type}+xml'
```

Comment:
Ah, right. I guess it doesn't hurt to add the v1, though I'm skeptical we'll ever have a v2. (I'm guessing we'd cali it something entirely different, to avoid confusion.) Will fix.